### ISSUE TITLE
Parsing phone numbers not working

### ISSUE THREAD

**Issue by bsarmiento-akurey** [marmelroy/PhoneNumberKit#229]
Every phone number I tried to parse keeps giving me an "invalidCountryCode" error. This was working 3 weeks ago perfectly.

**Comment by marmelroy** [comment]
@bsarmiento-akurey - sorry for the delay, do you have more details? did you test on device?

**Comment by kuyazee** [comment]
Inactive issue

**Comment by jdanthinne** [comment]
I've had this issue when trying to use PhoneNumberKit with SPM, but none when installed with Cocoapods. After some research, the problem was that the SPM version doesn't import `CoreTelephony.framework`, which PhoneNumberKit relies on…

**Comment by demosthese** [comment]
This is still an issue, PhoneNumberKit doesn't work at all with SPM.

**Comment by slxl** [comment]
this issue is caused by:

```
    public static func defaultMetadataCallback() throws -> Data? {
        let frameworkBundle = Bundle(for: PhoneNumberKit.self)
        guard let jsonPath = frameworkBundle.path(forResource: "PhoneNumberMetadata", ofType: "json") else {
            throw PhoneNumberError.metadataNotFound
        }
        let data = try Data(contentsOf: URL(fileURLWithPath: jsonPath))
        return data
    }
```
which throws `PhoneNumberError.metadataNotFound` bcs of missing json file. 

I'm wondering if there is a single person who made it work with SPM?)

**Comment by slxl** [comment]
just as a reference 
https://stackoverflow.com/questions/39815054/how-to-include-assets-resources-in-a-swift-package-manager-library

**Comment by kachod** [comment]
The workaround I did was to copy the `PhoneNumberMetadata.json` in the main bundle.

**Comment by BrentMifsud** [comment]
I think the workaround @kachod mentioned should be included in the readme. At least until SPM adds support for bundled assets.

**Comment by igorkulman** [comment]
This could be finally solved with Swift 5.3 https://github.com/apple/swift-evolution/blob/master/proposals/0271-package-manager-resources.md

**Comment by BrentMifsud** [comment]
Yep Ted Kremenek said it’s going to be in 5.3. So hopefully September/oct timeframe
